### PR TITLE
Add rank1 tests for bilinear, scale and complex variants

### PR DESCRIPTION
**Pull request by cgmillette**
- Add new yaml files for m1n1k1 tests

### CODE REVIEW

**Review by CongMa13 — DISMISSED** [review]
lgtm

**Review by dlangbe — DISMISSED** [review]

**Review by mkarunan — DISMISSED** [review]
LG

**Review by CongMa13 — CHANGES_REQUESTED** [review]
I noticed that hip does not define is_floating_point_v for bf16

`std::is_floating_point_v<bf16>  == false`
`std::is_floating_point_v<_Float16 >  == true`

File: test/device/common.hpp [hunk 1]
```diff
@@ -30,7 +30,18 @@
 template <typename T>
 __device__ inline double toDouble(T const& val)
 {
-    return static_cast<double>(static_cast<float>(val));
+    using hiptensor::float32_t;
+    using hiptensor::float64_t;
+
+    // Small floating point types need to convert up to single before double.
+    if constexpr((sizeof(T) < sizeof(float32_t)) && std::is_floating_point_v<T>)
```
Comment (CongMa13) [inline test/device/common.hpp]: I noticed that hip does not define is_floating_point_v for bf16

`std::is_floating_point_v<bf16>  == false`
`std::is_floating_point_v<_Float16 >  == true`

Comment (cgmillette) [reply test/device/common.hpp]: Fixed

**Review by CongMa13 — APPROVED** [review]
lgtm

**Pull request merged**